src/components/Error/Error.tsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+import cn from 'classnames';
+import { useEffect, useState } from 'react';
+
+type Props = {
+  error: string,
+};
+
+export const ShowError: React.FC<Props> = ({ error }) => {
```

Comment:
keep component Name & the file name synced :)
```suggestion
export const Error: React.FC<Props> = ({ error }) => {
```